src/components/Person/Person.jsx

```diff
@@ -1 +1,29 @@
-// export const Person = ({ person }) => ();
+/* eslint-disable no-nested-ternary */
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  const partnerSex = sex === 'f'
+    ? 'husband'
+    : 'wife';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+      {age && (
+        <p className="Person__age">{`I am ${age}`}</p>
+      )}
+      {isMarried
+        ? <p className="Person__partner">{`${partnerName} is my ${partnerSex}`}</p>
+        : <p className="Person__partner">I am not married</p>
+      }
```

Comment:
If you use a condition for content you can avoid repeating the 'p' ))

Reply:
Thx, you are right